src/new-client/src/App/Trades/TradesHeader/TradesHeader.tsx

```diff
@@ -30,17 +47,49 @@ const HeaderToolbar = styled("div")`
   align-items: center;
   justify-content: flex-end;
 `
-
-export const TradesHeader: React.FC = () => (
-  <TradesHeaderStyle>
-    <HeaderLeftGroup>Trades</HeaderLeftGroup>
-    <HeaderRightGroup>
-      <ExcelButton />
-      <HeaderToolbar>
-        <AppliedFilters />
-        <QuickFilter />
-      </HeaderToolbar>
-      <Fill />
-    </HeaderRightGroup>
-  </TradesHeaderStyle>
-)
+var isTornOut = false
+export const TradesHeader: React.FC = () => {
+  const tearOutTileState = useTearOutTileState("Tiles")
+  const tearOutAnalyticsState = useTearOutAnalyticsState("Analytics")
+  useObservableSubscription(
+    tearOutEntryTrades$.subscribe(async ([tornOut]) => {
+      if (tornOut && !isTornOut) {
+        isTornOut = true
+        openWindow(
+          {
+            url: constructUrl("/trades"),
+            width: window.innerWidth * 0.85,
+            name: "",
+            height: window.innerHeight * 0.25,
+          },
+          () => {
+            isTornOut = false
+            tearOutTrades(false)
+          },
+        )
+      }
+    }),
+  )
+  return (
+    <TradesHeaderStyle>
+      <HeaderLeftGroup>Trades</HeaderLeftGroup>
+      <HeaderRightGroup>
+        <ExcelButton />
+        <HeaderToolbar>
+          <AppliedFilters />
+          <QuickFilter />
+        </HeaderToolbar>
+        {!isTornOut && !(tearOutTileState && tearOutAnalyticsState) && (
+          <HeaderAction
+            onClick={() => {
+              tearOutTrades(true)
+            }}
+          >
+            <PopOutIcon />
```

Comment:
Maybe still on your list to do, but in case you overlooked - Need to change this to `PopInIcon` when it is popped out